src/services/__tests__/bitcoin.test.js

```diff
@@ -181,13 +181,13 @@ describe('bitcoin service', () => {
 
   describe('rootFromMnemonic', () => {
     it('returns a root node', async () => {
-      const root = await rootFromMnemonic(mnemonic);
+      const root = await rootFromMnemonic(mnemonic, 'testnet');
 
       expect(keyPairAddress(root)).toEqual('mhQ51TfiyTdwxYDq42Wrz1LvLY7PbSEJ9w');
     });
 
     it('can be used to derive an address', async () => {
-      const root = await rootFromMnemonic(mnemonic);
+      const root = await rootFromMnemonic(mnemonic, 'testnet');
       const keyPair = root.derivePath("m/49'/1/0");
       const derivedAddress = keyPairAddress(keyPair);
```

Comment:
Just a note that this will be gone soon - is it worth removing these tests now?

Reply:
just a big nuisance was that it failed tests due to prod env build and validation. Will be gone soon too